distributed/scheduler.py

```diff
@@ -2799,13 +2800,17 @@ def worker_objective(self, ts: TaskState, ws: WorkerState) -> tuple:
         """
         dts: TaskState
         comm_bytes: int = 0
+        xfers = 0
         for dts in ts.dependencies:
             if ws not in dts.who_has:
                 nbytes = dts.get_nbytes()
-                comm_bytes += nbytes
+                # amortize transfer cost over all waiters
+                comm_bytes += nbytes / len(dts.waiters)
+                xfers += 1
```

Comment:
```suggestion
                xfers += 1 / len(dts.waiters)
```
@gjoseph92 do you agree?